SAMtags.tex

```diff
@@ -253,6 +253,29 @@ \subsection{Metadata}
 \end{description}
 
 \subsection{Barcodes}
+DNA barcodes can be used to identify the provenance of the underlying reads.
+There are currently three varieties of barcodes that may co-exist: Sample Barcode, Cell Barcode, and Unique Molecular Identifier (UMI).
+\begin{itemize}
+\item
+Despite its name, the {\it Sample} Barcode identifies the {\it Library} and allows multiple libraries to be combined and sequenced together.
```

Comment:
Needs italic correction: `{\it Sample Barcode\/}` or more modernly `\emph{Sample Barcode}`.
And put _Barcode_ in italics too, and similarly for _Cell Barcode_ and _UMI_ below.